### DISCUSSION TITLE
Infrastructure Consultant Opportunity --> Please move this to https://jobs.cncf.io/. Recruiting is not allowed in this Github repo.

### DISCUSSION THREAD

**Discussion by kirstyleekay** [cncf/telecom-user-group#95] [General]
**Happy February 24th!** My team is looking for an Infrastructure expert to join our team in Southeast Michigan. If interested, feel free to reach out to me directly. More details here --> [https://plantemoran.avature.net/careers/JobDetail/Southfield-Michigan-United-States-Network-Infrastructure-Manager/5190]

**Comment by xmulligan** [comment]
Please move this to https://jobs.cncf.io/. Recruiting is not allowed in this Github repo.

**Reply by taylor** [reply]
@kirstyleekay Please move. We will be deleting this post soon.
